test/NFTOpt.tests.ts

```diff
@@ -177,10 +177,18 @@ describe("NFTOpt Tests", function () {
 
     describe("withdrawOptionRequest", function () {
         it("should test that method can be called", async function () {
-            expect(NFTOptCTR.connect(buyer)
-                            .withdrawOptionRequest(0))
-            .to.not.throw;
-        })
+            expect(
+                NFTOptCTR.connect(buyer).withdrawOptionRequest(0)
+            ).to.not.throw;
+        });
+
+        it("should fail when caller is not the buyer", async function () {
+            expect(
+                NFTOptCTR.connect(seller).withdrawOptionRequest(0)
+            ).to.not.throw;
+        });
+
+
     });
```

Comment:
suggest you start by withdrawing option 1, which you know doesn't exist, and should fail
publish an option request, then checking that you can withdraw that, which should succeed
next, you can try withdrawing the option with a different buyer, which should fail